src/linter/messages.ts

```diff
@@ -144,6 +151,22 @@ export const MESSAGE_INFO = {
 			`{@link sap.ui.core.IAsyncContentCreation sap.ui.core.IAsyncContentCreation}`,
 	},
 
+	[MESSAGE.MANIFEST_ASYNC_FALSE_ERROR]: {
+		severity: LintMessageSeverity.Error,
+		ruleId: RULES["async-component-flags"],
+
+		message: ({asyncFlagLocation}: {asyncFlagLocation: string}) =>
+			`The 'async' property at '${asyncFlagLocation}' must be removed`,
+		details: ({hasAsyncInterface}: {hasAsyncInterface?: boolean}) =>
+			`Setting 'async' to 'false' prevents asynchronous loading and is not recommended. Remove the 'async' flag` +
```

Comment:
At least in the context of the new checks, "not recommended" is too weak. it's not supported.